src/editors/supportedEditors.js

```diff
@@ -1,6 +1,7 @@
 import TextEditor from './containers/TextEditor';
 import VideoEditor from './containers/VideoEditor';
 import ProblemEditor from './containers/ProblemEditor';
+import VideoUploadEditor from './containers/VideoUploadEditor';
```

Comment:
nit: I don't know what qualifies as "ADDED_EDITORS", but since in the `supportedEditors` object the new editor was added below the comment, to be consistent, the corresponding import should be below the comment on line 6 as well.

Reply:
Ack. Moved it.